pkg/interface/soto/src/js/components/history.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from 'react';
+
+export class History extends Component {
+    constructor(props) {
+        super(props);
+    }
+
+render() {
+    return (
+        <div className="history flex flex-column-reverse overflow-container"
+        style={{ height: 'calc(100% - 1rem)', resize: 'vertical' }}>
```

Comment:
This looks similar to what chat-web does. Does this scroll in Firefox?

Reply:
I did test in Firefox. It stays scrolled at bottom, no scroll, same as Chat’s behaviour, even if the container begins to overflow. (@loganallenc casually said a terminal was a chat room, so I did likewise!)